docs/docs/seo.md

```diff
@@ -7,14 +7,23 @@ Gatsby can help your site rank and perform better in search engines. Using Gatsb
 ## Server rendering
 
 Because Gatsby pages are server-side rendered, all the page content is available to Googlebot and other search engine crawlers.
+You can see this by viewing the source for this page in your browser Right-Click => View source. You'll see the fully rendered HTML document.
 
-You can see this by viewing the source for this page with `curl` (in your terminal):
+When you've installed [`gatsby-plugin-offline`](https://www.gatsbyjs.org/packages/gatsby-plugin-offline/), you'll see a partial HTML document that does not contain the HTML you were hoping for. We enable this trick to optimize bandwidth consumption and not let your users download too much data. Serving a partial HTML document is okay. Google and other search engines will still see the full HTML because `gatsby-plugin-offline` only starts working on the second-page load. A search engine always runs a page in Sandbox mode, which essentially is the first visit.
+
+As a website owner, how do I test my site is serving it's HTML correctly when `gatsby-plugin-offline` is being used? It would be best if you used your terminal of choice to visit your website. You can crawl your site by running the following command:
```

Comment:
```suggestion
As a website owner, how do I test my site is serving its HTML correctly when `gatsby-plugin-offline` is being used? It would be best if you used your terminal of choice to visit your website. You can crawl your site by running the following command:
```

Reply:
It's not really a "trick" - maybe use something like this instead: 

Old:`We enable this trick to optimize bandwidth consumption and not let your users download too much data.`
Proposed: `By registering and using a service worker, we can optimize bandwidth consumption and not let your users download too much data.`

Thoughts?

Reply:
That's a good point tcope. I'll rewrite it a bit (unsure if the terminology service worker should be mentioned here)